src/lib.rs

```diff
@@ -97,25 +98,33 @@
 //!
 //! Check out the [custom_key](https://github.com/AaronErhardt/actix-governor/blob/main/examples/custom_key.rs) example to see how a custom key extractor can be implemented.
 //!
-//! # Customize response error content
+//! # Customize responses
+//! There are two errors that you can customize, they are the error that the request limit is exceeded and
+//! the second is the error that the keys cannot be extracted, let's see how you can customize them
```

Comment:
I'd change the text a bit and also the order of the errors because the extraction error occurs first in the middleware.

```markdown
# Customizing error responses

There are two errors that might occur during rate-limiting.
The first error occurs when the key can't be extracted from the request, for example when a session id is missing.
The second error occurs when the rate limit is exceeded.
```